topostats/plottingfuncs.py

```diff
@@ -89,7 +108,17 @@ def plot_and_save(
 
         if save:
             if image_set == "all" or core_set:
-                plt.savefig(output_dir / filename)
+                if not axes and not colorbar:
+                    plt.title('')
+                    fig.frameon=False
+                    plt.savefig(
+                        (output_dir / filename).with_suffix('.'+save_format), 
```

Comment:
Probably think I'm crazy banging on about f-strings but is there any reason not to use one here?
```suggestion
                        (output_dir / filename).with_suffix(f".{save_format}"), 
```

Reply:
Apologies, just not used to using them yet